# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/res/forms.css
# Commit: 0186165ee1d8
# Full Hash: 0186165ee1d84f56362ba16bbda4eefa1d47fd60
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2019-11-26 05:19:55
# Regressor Bug: 471015
# File Overlap Count: 1
# Description:
#   Bug 471015 - [css-break] Implement <fieldset> fragmentation.  r=TYLin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D53709
# ==============================================================================

diff -r 9cf1851c76dc -r 0186165ee1d8 layout/style/res/forms.css
--- a/layout/style/res/forms.css	Mon Nov 25 15:47:38 2019 +0000
+++ b/layout/style/res/forms.css	Mon Nov 25 18:05:16 2019 +0000
@@ -19,6 +19,7 @@
      borders we don't leak out the corners for hit-testing purposes. */
   border-radius: inherit;
   padding: inherit;
+  box-decoration-break: inherit;
   block-size: 100%; /* Need this so percentage block-sizes of kids work right */
   /* Please keep the Multicol/Flex/Grid/Align sections below in sync with
      ::-moz-scrolled-content in ua.css and ::-moz-button-content below. */